pkg/services/supportbundles/supportbundlesimpl/service.go

```diff
@@ -49,18 +52,21 @@ func ProvideService(cfg *setting.Cfg,
 	pluginSettings pluginsettings.Service,
 	features *featuremgmt.FeatureManager,
 	usageStats usagestats.Service) (*Service, error) {
+	section := cfg.SectionWithEnvOverrides("support_bundles")
 	s := &Service{
-		cfg:            cfg,
-		store:          newStore(kvStore),
-		pluginStore:    pluginStore,
-		pluginSettings: pluginSettings,
-		accessControl:  accessControl,
-		features:       features,
-		log:            log.New("supportbundle.service"),
-		collectors:     make(map[string]supportbundles.Collector),
+		cfg:             cfg,
+		store:           newStore(kvStore),
+		pluginStore:     pluginStore,
+		pluginSettings:  pluginSettings,
+		accessControl:   accessControl,
+		features:        features,
+		log:             log.New("supportbundle.service"),
+		enabled:         section.Key("enabled").MustBool(true),
```

Comment:
I guess we are going to remove the feature toggle when ready but still have a way to enable / disable this feature?

Reply:
yes, in case we want to disable the feature